parser/sql/dialect/opengauss/src/main/antlr4/imports/opengauss/BaseRule.g4

```diff
@@ -494,6 +495,10 @@ name
     : identifier
     ;
 
+modelName
```

Comment:
Can you add test case for this statement?

Reply:
sure